packages/next/src/cli/next-dev.ts

```diff
@@ -403,7 +404,11 @@ If you cannot make the changes above, but still want to try out\nNext.js v13 wit
       startServer(devServerOptions)
         .then(async (app) => {
           const appUrl = `http://${app.hostname}:${app.port}`
```

Comment:
We don't need to change this?

Reply:
Great. lets <picture data-single-emoji=":shipit:" title=":shipit:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..wVxnRUcaGpoyL4p7.1KKOM9OBc5yzVReOm2XStRYsk7z9PDeqRuEzKnIugIy4LPhYC0Dm_07FhdV7DNujGjQ5Y84I9rzzAc2hScc9GFP8OIp3TiTzOhIN-027XKReIA.5axmcXEYGzOfekFt3Ei1kg" alt=":shipit:" align="absmiddle"></picture>